guides/Dataset.md

```diff
@@ -684,11 +711,12 @@ A spatial reference system can be added in this way:
     "@type": "Place",
     "geo": {
       "@type": "GeoShape",
-      "line": "39.3280,120.1633 40.445,123.7878"
+      "line": "39.3280 120.1633 40.445 123.7878"
     },
     <strong>"additionalProperty": {
-      "@type": ["PropertyValue", "dbpedia:Spatial_reference_system"],
-      "@id": "http://www.opengis.net/def/crs/OGC/1.3/CRS84"
+      "@type": "PropertyValue",
+      "propertyID":"dbpedia:Spatial_reference_system",
```

Comment:
for here, we may want to use the full URL or change the value to use `@id` so it knows its a resource and not a string: ```{ "@id": "dbpedia:Spatial_reference_system" },```

@smrgeoinfo , do you agree?

Reply:
Sorry, I don't agree.  @id identifies the object in the schema.org document, not what the PropertyValue is. Its like a document fragment identifier.   The @type types the object that is the value of the schema:additionalProperty; in this context I think 'type' refers to the properties that are expected in that value (a PropertyValue). We're not adding any properties related to a dbpedia:Spatial_reference_system, so its not needed.

We're hamstrung by schema.org's looseness about what a schema:URL value is supposed to be.

Reply:
I did change the dbpedia: URI to the full HTTP URI value.

Reply:
ok great! just wanted to make sure that if we wanted the prefixed uris to be expanded, we enabled that!